### DISCUSSION TITLE
Can we use in Nuxt 2 / Vue 2?

### DISCUSSION THREAD

**Discussion by akshaysinghtk14** [ndabAP/vue-command#432] [Q&A]
Can we use in Nuxt 2 / Vue 2 ? @ndabAP

**Comment by ndabAP** [comment]
Yes, in older versions. But it looked different 😉. Check the commit history if you need a fast answer. I can only look later.

**Reply by ndabAP** [reply]
`v23.0.1` is the last compatible version for Vue.js 2.

**Reply by akshaysinghtk14** [reply]
@ndabAP Can we access latest feature vue-command in v23.0.1 ?

**Reply by ndabAP** [reply]
> @ndabAP Can we access latest feature vue-command in v23.0.1 ?

No, only with the newest version.
